RoosterPlanner.Data/Common/Repository.cs

```diff
@@ -1,14 +1,11 @@
 ﻿using System;
```

Comment:
Deze changes zijn niet relevant voor deze PR en dus verwarrend om te reviewen. Probeer deze te splitsen in een andere geature branch